sdk/datafactory/Azure.ResourceManager.DataFactory/tests/Scenario/LinkedServiceResourceTests.cs

```diff
@@ -10,30 +10,44 @@
 using Azure.ResourceManager.DataFactory.Models;
 using Azure.ResourceManager.DataFactory.Tests;
 using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Storage;
 using NUnit.Framework;
 
 namespace Azure.ResourceManager.IotHub.Tests.Scenario
 {
     internal class LinkedServiceResourceTests : DataFactoryManagementTestBase
     {
-        private const string _accessKey = "REDACTED";
+        private string _accessKey;
+        private ResourceGroupResource _resourceGroup;
         private DataFactoryResource _dataFactory;
         public LinkedServiceResourceTests(bool isAsync) : base(isAsync)
         {
+            JsonPathSanitizers.Add("$.keys.[*].value");
```

Comment:
I saw this line of code is also used in `DataFactoryDatasetResourceTests`. Therefore I will suggest you removing this to the ctor of the base class to avoid the duplicate.